tests/e2e/steps.go

```diff
@@ -106,3 +100,18 @@ var changeoverSteps = concatSteps(
 
 	stepsPostChangeoverDelegate("sover"),
 )
+
+var consumerMisbehaviourSteps = concatSteps(
+	// start provider and consumer chain
+	stepsStartChainsWithSoftOptOut("consu"),
+	// make a consumer validator to misbehave and get jailed
+	stepsCauseConsumerMisbehaviour("consu"),
+)
+
+var consumerDoubleSignSteps = concatSteps(
+	// start provider and consumer chain
+	stepsStartChains([]string{"consu"}, false),
+
```

Comment:
nit: Why do we have a new line here while in  `consumerMisbehaviourSteps` we do not have a line between `stepsStartChainsWithSoftOptOut("consu")` and `stepsCauseConsumerMisbehaviour("consu")`.